api.web.yaml

```diff
@@ -610,6 +611,9 @@ components:
           items:
             type: string
           description: List of additional study administrator usernames (empty array if none)
+        has_existing_data:
+          type: boolean
+          description: Whether the study involves existing data (e.g., data already collected or available)
```

Comment:
This could alternatively be a string if other kinds of answers are allowable

Reply:
think a bool makes most sense cause either there is data or not, right?